lib/AddDependencies.js

```diff
@@ -122,7 +125,7 @@ class AddDependencies {
   }
 
   saveToPackage() {
-    const packageFilePath = 'package.json';
+    const packageFilePath = this.packageDir;
```

Comment:
This variable is redundant since we have `this. packageFilePath` (see comments above).

Reply:
addressed